home/src/main/resources/rdf/tbox/firsttime/classGroupAnnotationsVitro.n3

```diff
@@ -0,0 +1,85 @@
+		vitro:hiddenFromDisplayBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+		vitro:hiddenFromPublishBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+		vitro:prohibitedFromUpdateBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
+search:SearchField 
+		vitro:inClassGroup <http://vivoweb.org/ontology#vitroClassGroupsearchsettings> ;
+		vitro:hiddenFromDisplayBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+		vitro:hiddenFromPublishBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+		vitro:prohibitedFromUpdateBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
+search:FilterValue
+		vitro:inClassGroup <http://vivoweb.org/ontology#vitroClassGroupsearchsettings> ;
+		vitro:hiddenFromDisplayBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+		vitro:hiddenFromPublishBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+		vitro:prohibitedFromUpdateBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
+search:Statement 
+        vitro:inClassGroup <http://vivoweb.org/ontology#vitroClassGroupsearchsettings> ;
+        vitro:hiddenFromDisplayBelowRoleLevelAnnot     <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:hiddenFromPublishBelowRoleLevelAnnot     <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:prohibitedFromUpdateBelowRoleLevelAnnot  <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
+search:Filter
+        vitro:inClassGroup <http://vivoweb.org/ontology#vitroClassGroupsearchsettings> ;
+        vitro:hiddenFromDisplayBelowRoleLevelAnnot     <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:hiddenFromPublishBelowRoleLevelAnnot     <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:prohibitedFromUpdateBelowRoleLevelAnnot  <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
+search:FilterGroup 
+    	vitro:inClassGroup <http://vivoweb.org/ontology#vitroClassGroupsearchsettings> ;
+        vitro:hiddenFromDisplayBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:hiddenFromPublishBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:prohibitedFromUpdateBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
+search:Sort 
+    	vitro:inClassGroup <http://vivoweb.org/ontology#vitroClassGroupsearchsettings> ;
+        vitro:hiddenFromDisplayBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:hiddenFromPublishBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> ;
+        vitro:prohibitedFromUpdateBelowRoleLevelAnnot <http://vitro.mannlib.cornell.edu/ns/vitro/role#public> .
+
```

Comment:
do we still need those annotations, even though it is available for anyone? I was thinking this is default behaviour (visible  for anyone). Also, why Publish and Update are specified here?

Reply:
I am not sure it should be visible for anyone as those are configuration properties. Maybe we should allow only admins to display/publish/update those settings?

Reply:
This file was removed as search configurations were moved into display model.